spec/controllers/users_controller_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+RSpec.describe UsersController, type: :controller do
+
+  let(:users) { create_list(:user, 3) }
+  let(:user) { users[0] }
+
+  describe 'GET #index' do
+    before { get :index }
+
+    it 'populates an array of all users' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.